device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-D36/1/port_config.ini

```diff
@@ -0,0 +1,21 @@
+# name         lanes                             alias        index  role       speed       asic_port_name     coreId      corePortId      numVoq
+Ethernet144    72,73,74,75,76,77,78,79           Ethernet19/1 19     Ext        400000      Eth144-ASIC1       1           1               8
+Ethernet152    80,81,82,83,84,85,86,87           Ethernet20/1 20     Ext        400000      Eth152-ASIC1       1           3               8
+Ethernet160    88,89,90,91,92,93,94,95           Ethernet21/1 21     Ext        400000      Eth160-ASIC1       1           5               8
+Ethernet168    96,97,98,99,100,101,102,103       Ethernet22/1 22     Ext        400000      Eth168-ASIC1       1           7               8
+Ethernet176    104,105,106,107,108,109,110,111   Ethernet23/1 23     Ext        400000      Eth176-ASIC1       1           9               8
+Ethernet184    112,113,114,115,116,117,118,119   Ethernet24/1 24     Ext        400000      Eth184-ASIC1       1           11              8
+Ethernet192    120,121,122,123,124,125,126,127   Ethernet25/1 25     Ext        400000      Eth192-ASIC1       1           13              8
+Ethernet200    128,129,130,131,132,133,134,135   Ethernet26/1 26     Ext        400000      Eth200-ASIC1       1           15              8
+Ethernet208    136,137,138,139,140,141,142,143   Ethernet27/1 27     Ext        400000      Eth208-ASIC1       1           17              8
+Ethernet216    64,65,66,67,68,69,70,71           Ethernet28/1 28     Ext        400000      Eth216-ASIC1       0           19              8
+Ethernet224    56,57,58,59,60,61,62,63           Ethernet29/1 29     Ext        400000      Eth224-ASIC1       0           21              8
+Ethernet232    48,49,50,51,52,53,54,55           Ethernet30/1 30     Ext        400000      Eth232-ASIC1       0           23              8
+Ethernet240    40,41,42,43,44,45,46,47           Ethernet31/1 31     Ext        400000      Eth240-ASIC1       0           25              8
+Ethernet248    32,33,34,35,36,37,38,39           Ethernet32/1 32     Ext        400000      Eth248-ASIC1       0           27              8
+Ethernet256    24,25,26,27,28,29,30,31           Ethernet33/1 33     Ext        400000      Eth256-ASIC1       0           29              8
+Ethernet264    16,17,18,19,20,21,22,23           Ethernet34/1 34     Ext        400000      Eth264-ASIC1       0           31              8
+Ethernet272    8,9,10,11,12,13,14,15             Ethernet35/1 35     Ext        400000      Eth272-ASIC1       0           33              8
+Ethernet280    0,1,2,3,4,5,6,7                   Ethernet36/1 36     Ext        400000      Eth280-ASIC1       0           35              8
+Ethernet-Rec1  221                               Recirc0/0    19     Rec        400000      Rcy0-ASIC1         0           221             8
```

Comment:
The port index for the `Rec1`  and `IB1` overlap with the `Ethernet144` and `Ethernet152`